citesphere/src/main/webapp/WEB-INF/views/auth/personalAccessToken.html

```diff
@@ -0,0 +1,40 @@
+<html layout:decorate="~{layouts/main}">
+<head>
+<script th:src="@{/resources/notify/bootstrap-notify.min.js}"></script>
+<link th:href="@{/resources/notify/animate.css}" rel="stylesheet" />
+</head>
+
+<body>
+	<div layout:fragment="content">
+		<p>
+			<a th:href="@{/auth/savePersonalAccessToken}"
+				class="btn btn-primary btn-sm" id="genPersonalToken"><i class="icon-circle-add"></i>
+				&nbsp;Generate New Token</a>
+		</p>
+		<div class="col-md-10">
+			<div>
+				<h4>[[${personalTokensCount}]] tokens for this user:</h4>
+			</div>
+			<table class="table table-striped table-bordered">
+				<tr>
+					<th>User</th>
+					<th>Token</th>
+					<th>Creation Date</th>
+				</tr>
+
+				<tr th:id="'tr-'+${entry.id}" data-action='draggable' class="items"
+					th:each="entry,loop : ${listOfPersonalTokens}">
+					<td th:data-index="${loop.index}"><strong style="color: black">[[${entry.username}]]</strong>
+					</td>
+					<td th:data-index="${loop.index}"><strong style="color: black">[[${entry.token}]]</strong>
```

Comment:
Tokens should not be shown on the front end (they actually couldn't be shown anyways, since they are stored encryped)